barretenberg/ts/src/barretenberg_api/index.ts

```diff
@@ -545,6 +545,18 @@ export class BarretenbergApi {
     return out as any;
   }
 
+  async acirProveAndVerifyAztecClient(acirVec: Uint8Array[], witnessVec: Uint8Array[]): Promise<boolean> {
```

Comment:
We don't need separate prove and verify flows yet. My goal is to get something measurable, someone can easily follow the pattern to add these in the future.

Reply:
Also we don't ever want a verify flow through wasm. Prob we should star to cut stuff out of the wasm binary since it's already large.